site/pages/react/ui-components.mdx

```diff
@@ -83,42 +97,20 @@ The easiest way to configure your UI components, is using the [Demo App](https:/
 - `sections` object is an array of arrays representing the various sections in descending order, each section (element in the outer array) separated by a divider in the UI
 - each item within a section is an `AuthTypes` objects containing configuration parameters for that authentication method.
 
-  <br />
-
 **The following [`AuthTypes`](https://github.com/alchemyplatform/aa-sdk/blob/main/account-kit/react/src/components/auth/types.ts)
 can be passed into sections:**
 
 ### Email
 
-This adds an email login to a section. The shape of this auth type is:
-
-```ts
-type EmailAuthType = {
-  type: "email";
-  // hides the continue button
-  hideButton?: boolean;
-  // changes the button label
-  buttonLabel?: string;
-  // changes the placeholder text in the input
-  placeholder?: string;
-};
-```
+This adds an email login to a section. For detailed information on email authentication, see [Email OTP](/react/login-methods/email-otp) and [Email Magic Link](/react/login-methods/email-magic-link) pages.
 
 ### Passkey
 
-This adds a passkey login method to a section. Passkey login allows users to log in with a passkey they've created else where in your app after logging in, or right after sign-up if you've enabled that setting.
-The shape of this auth type is:
-
-```ts
-type PasskeyAuthType = {
-  type: "passkey";
-};
-```
+This adds a passkey login method to a section. For detailed information on passkey authentication, see [Passkey Login](/react/login-methods/passkey-login) and [Passkey Signup](/react/login-methods/passkey-signup) pages.
 
 ### External wallets
 
 This adds an external wallet login method to a section. This allows users to connect to your app with existing EOAs via browser extension or WalletConnect.
-The shape of this auth type is:
 
 ```ts
 type ExternalWalletsAuthType = {
```

Comment:
I was wondering if we should include external wallets as a login method.  This seems to be the only place we mention it